plugins/nginx-vhosts/commands

```diff
@@ -49,6 +49,9 @@ case "$1" in
       shopt -u nullglob
     fi
 
+    DOKKU_APP_CIDS=($(get_container_ids $APP))
+    docker cp "${DOKKU_APP_CID[0]}:/app/nginx.conf" "$APP_NGINX_TEMPLATE" 2> /dev/null || true
```

Comment:
I really think we should to be consistent and to call out that the file is actually a template and not a raw conf file.